docroot/sites/all/themes/ilr_theme/js/ilr_subnav.js

```diff
@@ -93,7 +93,12 @@
           }
         });
         liHeight = $('#sidebar-first ul.menu > li').outerHeight();
-        height = height + 2 * liHeight; // There are two additional list items added
+        searchHeight = $('#block-ilr-sdc-listings-course-search').outerHeight();
+        if ($('body').hasClass('page-professional-programs-public-offerings')) {
```

Comment:
Thanks for this, John. However, the search/filter can be added as a block to any page, so you should probably check for the presence of the filter rather than the page class.